en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md

```diff
@@ -0,0 +1,139 @@
+# Adding Multiple Keys to the Primary Keystore
+
+WSO2 Identity Server enables adding multiple keys to the primary keystore. Let's explore the following topics to learn more.
```

Comment:
We will have to explain why we should maintain multiple keys and state whether this is only supported for super tenants. @inthirakumaaran please verify.